pom.xml

```diff
@@ -86,6 +86,7 @@
         <mockito.version>1.10.19</mockito.version>
         <hamcrest-all.version>1.3</hamcrest-all.version>
         <junit.version>4.12</junit.version>
+        <awaitility.version>3.0.0</awaitility.version>
```

Comment:
Do we expect this to only be an itests? There may have been a decision to keep these properties up in the root pom.